modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/AuditBarChart.js

```diff
@@ -132,49 +125,102 @@ export default function AuditBarChart({rtl, vocabularies}) {
 	}, [colors]);
 
 	const renderLegend = (props) => {
-		const {payload} = props;
+		const {payload, yAxisName} = props;
+
+		return (
+			<ClayLayout.ContainerFluid>
+				<ClayLayout.Row justify="start">
```

Comment:
Isn't the row "justify-start" by default? Unless you had it overwritten somewhere else I think you can omit `justify="start"` here
https://codepen.io/pen?template=vMewQz

Reply:
You're right. I'm going to remove it :)